typing/datatype.go

```diff
@@ -101,6 +103,21 @@ func ReformatValue(v interface{}) interface{} {
 	return interface{}(intValue)
 }
 
+// ReformatTimeValue processes string with ISO DateTime into time.Time
+func ReformatTimeValue(value interface{}) interface{} {
+	stringValue, ok := value.(string)
+	if !ok {
+		return value
+	}
+
+	timeValue, err := time.Parse(timestamp.Layout, stringValue)
```

Comment:
Since we're going to support both formats (`timestamp.Layout` and `timestamp.DeprecatedLayout`),
please add parsing `timestamp.DeprecatedLayout` as well. (consider rename DeprecatedLayout into something without `Deprecated`)

Reply:
My suggestion is to remove `DeprecatedLayout` at all.
Formatting timestamp is better for using only one `Layout` format.
Parsing is better to use from all format as `RFC3339Nano` - feel free to see `TestReformatTimeValue`

Reply:
@Alessar  It is a good idea. Then it would be better to refactor func `stringToTimestamp()` in `typing.Converter` - replace logic with `Layout` and `DeprecatedLayout` with your solution. Then `DeprecatedLayout` can be deleted.